build.gradle.kts

```diff
@@ -46,6 +46,7 @@ tasks {
 }
 
 dependencies {
+    implementation("jakarta.inject:jakarta.inject-api:1.0")
```

Comment:
What is this being used for?

Reply:
I added it for the `javax.inject.Provider` interface. You can see it in action in `DefaultResponseFormatter`, where I'm using it to avoid a circular dependency issue.